cmd/root.go

```diff
@@ -258,6 +258,9 @@ func setupEnv() (string, error) {
 
 	// Create snapshots dir if it doesn't exist
 	snapshotsDir := filepath.Join(baseDir, constants.SnapshotsDirName)
+	if utils.IsE2E() {
```

Comment:
probably we just want to change baseDir if IsE2E

Reply:
addressed. it is a better idea indeed, it allows us to separate other e2e related testing in future.